### ISSUE TITLE
eval PP-OCRv4_det warning

### ISSUE THREAD

**Issue by SophiaSarah** [PaddlePaddle/PaddleOCR#11960]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：
W0417 20:23:57.326244 170990 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 8.0, Driver API Version: 11.7, Runtime API Version: 11.6
W0417 20:23:57.331068 170990 gpu_resources.cc:91] device: 0, cuDNN Version: 8.4.
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.0.se_block.conv1.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.0.se_block.conv1.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.0.se_block.conv2.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.0.se_block.conv2.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.1.se_block.conv1.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.1.se_block.conv1.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.1.se_block.conv2.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.1.se_block.conv2.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.2.se_block.conv1.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.2.se_block.conv1.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.2.se_block.conv2.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.2.se_block.conv2.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.3.se_block.conv1.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.3.se_block.conv1.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.3.se_block.conv2.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.ins_conv.3.se_block.conv2.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.0.se_block.conv1.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.0.se_block.conv1.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.0.se_block.conv2.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.0.se_block.conv2.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.1.se_block.conv1.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.1.se_block.conv1.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.1.se_block.conv2.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.1.se_block.conv2.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.2.se_block.conv1.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.2.se_block.conv1.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.2.se_block.conv2.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.2.se_block.conv2.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.3.se_block.conv1.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.3.se_block.conv1.bias not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.3.se_block.conv2.weight not in loaded params!
[2024/04/17 20:24:01] ppocr WARNING: neck.inp_conv.3.se_block.conv2.bias not in loaded params!
[2024/04/17 20:24:04] ppocr INFO: resume from ./models/ch_PP-OCRv4_det_train/best_accuracy
[2024/04/17 20:24:04] ppocr INFO: metric in ckpt ***************
[2024/04/17 20:24:04] ppocr INFO: is_float16:False
eval model:: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 500/500 [00:41<00:00, 12.11it/s]
[2024/04/17 20:24:45] ppocr INFO: metric eval ***************
[2024/04/17 20:24:45] ppocr INFO: precision:0.4501108647450111
[2024/04/17 20:24:45] ppocr INFO: recall:0.3909484833895041
[2024/04/17 20:24:45] ppocr INFO: hmean:0.418448853388302
[2024/04/17 20:24:45] ppocr INFO: fps:15.757990794258408

**Comment by Sunting78** [comment]
您好，可以给下运行命令。pretrain_model换为训练好的模型，checkpoint也换为训练好的模型。再试试看

**Comment by SophiaSarah** [comment]
> 您好，可以给下运行命令。pretrain_model换为训练好的模型，checkpoint也换为训练好的模型。再试试看
python3 tools/eval.py -c configs/det/det_PP-OCRv4_db.yml
其中det_PP-OCRv4_db.yml内容如下
Global:
  use_gpu: true
  use_xpu: false
  use_mlu: false
  epoch_num: 1200
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/ch_PP-OCRv4/
  save_epoch_step: 1200
  # evaluation is run every 2000 iterations
  eval_batch_step: [0, 2000]
  cal_metric_during_train: False
  pretrained_model: null
  checkpoints: ./models/ch_PP-OCRv4_det_train/best_accuracy.pdparams
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt
  distributed: false

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: PPLCNetV3
    scale: 0.75
    pretrained: false
    det: true
  Neck:
    name: RSEFPN
    out_channels: 96
    shortcut: true
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.001
  regularizer:
    name: 'L2'
    factor: 0

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ***********
    label_file_list:
      - **********
    ratio_list: [1.0]
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - IaaAugment:
        augmenter_args:
        - type: Fliplr
          args:
            p: 0.5
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    - EastRandomCropData:
        size:
        - 640
        - 640
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7
        total_epoch: 500
    - MakeShrinkMap:
        shrink_ratio: 0.4
        min_text_size: 8
        total_epoch: 500
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - threshold_map
        - threshold_mask
        - shrink_map
        - shrink_mask
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 16
    num_workers: 8
    use_shared_memory: True
  sampler:
    name: BatchSampler
Eval:
  dataset:
    name: SimpleDataSet
    data_dir:*********
    label_file_list:
      - ***********
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest: null
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 8
    use_shared_memory: True
  sampler:
    name: BatchSampler

checkpoints为 https://paddleocr.bj.bcebos.com/PP-OCRv4/chinese/ch_PP-OCRv4_det_train.tar

**Comment by UserWangZz** [comment]
checkpoints: ./models/ch_PP-OCRv4_det_train/best_accuracy
这里不需要填写文件扩展名，可以试一下

**Comment by Sunting78** [comment]
您好配置文件请使用这个目录下的文件 https://github.com/PaddlePaddle/PaddleOCR/tree/main/configs/det/ch_PP-OCRv4 如果是轻量模型选择student，服务端选择teacher的配置文件

**Comment by HonestyBrave** [comment]
> 您好配置文件请使用这个目录下的文件 https://github.com/PaddlePaddle/PaddleOCR/tree/main/configs/det/ch_PP-OCRv4 如果是轻量模型选择student，服务端选择teacher的配置文件

请问，cml这个文件对应哪个预训练模型？
